lib/say_cases.rb

```diff
@@ -0,0 +1,46 @@
+class SayCase < OpenStruct
+  def test_name
+    'test_%s' % description.gsub(/[-,]/,'_').downcase.tr(' ', '_')
+  end
+
+  def workload
+    [
+      "question = #{underscore_format(input)}",
+      indent(4, assertion),
+    ].join("\n")
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  private
+
+  def indent(size, lines)
+    lines.lines.each_with_object('') { |line, obj| obj << ' ' * size + line }
+  end
+
+  def assertion
+    return error_assertion if expected == -1
+
+    "assert_equal('#{expected}', Say.new(question).in_english)"
+  end
+
+  def error_assertion
+    [
+      'assert_raises ArgumentError do',
+      indent(2, 'Say.new(question).in_english'),
+      'end',
+    ].join("\n")
+  end
+
+  def underscore_format(number)
+    number.to_s.reverse.gsub(/...(?=.)/,'\&_').reverse
```

Comment:
This line needs a space after the comma too.

You can fix both of these automatically by running `bundle exec rubocop -a`